README.md

```diff
@@ -163,8 +171,21 @@ There's also an option to delete assets from CDF that are not referenced in the
 | `subtrees` | `ingest` | Controls what should happen with subtrees without a root node in the input data. `ingest` says they will be processed and loaded into CDF, `ignore` will ignore all of them and `error` will stop the execution and raise an error (nothing will be ingested). |
 | `batchSize`                | 1000    | The number of assets to write per API call.                                                                       |
 
-### Example
+### Setup for Python
+
+You may want to set up a jupyter notebook with `pySpark` running.
+
+* Download spark version `2.4.5` [here](https://www.apache.org/dyn/closer.lua/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz)
+* Follow the instuctions given [here](https://www.sicara.ai/blog/2017-05-02-get-started-pyspark-jupyter-notebook-3-minutes) except that your spark version will be different than what is written over there.
+* Start your jupyter notebook with the following command instead of simple `pyspark` (as written in the above link):
```

Comment:
```suggestion
You may want to set up a jupyter notebook with `pySpark` running.

* Download spark version `2.4.5` [here](https://www.apache.org/dyn/closer.lua/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz)
* Follow the instructions given [here](https://www.sicara.ai/blog/2017-05-02-get-started-pyspark-jupyter-notebook-3-minutes), except that your Spark version will be `2.4.5`.
* Start your Jupyter notebook with the following command (instead of `pyspark` as in the link above):
```